packages/twenty-server/src/modules/workflow/workflow-step-executor/factories/send-email-action-executor.factory.ts

```diff
@@ -0,0 +1,77 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import { z } from 'zod';
+import Handlebars from 'handlebars';
+import { JSDOM } from 'jsdom';
+import DOMPurify from 'dompurify';
+import { WorkflowActionEmail } from 'twenty-emails';
+import { render } from '@react-email/components';
+
+import { WorkflowStepResult } from 'src/modules/workflow/common/types/workflow-step-result.type';
+import { WorkflowSendEmailStep } from 'src/modules/workflow/common/types/workflow-step.type';
+import { EnvironmentService } from 'src/engine/integrations/environment/environment.service';
+import { EmailService } from 'src/engine/integrations/email/email.service';
+
+@Injectable()
+export class SendEmailActionExecutorFactory {
```

Comment:
this is not a factory to me : a factory is providing an implementation of an abstract service.
I feel we are over-using this factory naming in the backend codebase

Reply:
yes, I felt the same, but it is done like that everywhere else in the project, so I did the same

Reply:
New naming proposal for factories:
<img width="351" alt="image" src="https://github.com/user-attachments/assets/7fe904ef-6762-4405-8f11-10bd57959fa0">

Reply:
`workflow-step-executors` instead of `factories` actually